code/modules/mob/living/carbon/human/human.dm

```diff
@@ -894,9 +894,14 @@
 	if(istype(location, /turf))
 		location.add_vomit_floor(src, 1)
 
+	if(reagents.reagent_list.len)
+		var/purge_percent = 0.7 //30%
```

Comment:
what's the point of making this a local var if you're using it nowhere else but line 900

Reply:
Originally I was just exposing vars for ease of seeing what value was being used (I thought I needed three, but over time managed to whittle it down to one) I'll just set line 900 to * 0.7